akka-http2-support/src/main/scala/akka/http/impl/engine/http2/FrameEvent.scala

```diff
@@ -14,6 +14,7 @@ sealed trait StreamFrameEvent extends FrameEvent {
   def streamId: Int
 }
 
+final case class GoAwayFrame(lastStreamId: Int, errorCode: ErrorCode, debug: Option[String] = None) extends FrameEvent
```

Comment:
ByteString instead of String for the debug data? I don't see in the spec where it said that it's a string.